src/pages/Main/NewProducts.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './NewProducts.scss';
+
+export default function NewProducts({
+  productName,
+  productPrice,
+  productImg,
+  imgAlt,
+  productPage,
+}) {
+  return (
+    <div className="newProductBox">
```

Comment:
```suggestion
    <div className="newProducts">
```
컴포넌트의 이름과 동일하게 해주세요!